src/main/java/mate/academy/spring/validation/PasswordValidator.java

```diff
@@ -0,0 +1,15 @@
+package mate.academy.spring.validation;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import mate.academy.spring.dto.request.UserRequestDto;
+
+public class PasswordValidator implements ConstraintValidator<ValidPassword, UserRequestDto> {
+    @Override
+    public boolean isValid(UserRequestDto userRequestDto,
+                           ConstraintValidatorContext constraintValidatorContext) {
+        return (userRequestDto.getPassword() != null && userRequestDto.getRepeatPassword() != null)

```

Comment:
```suggestion
        return userRequestDto.getPassword() != null && 

```

Reply:
it not clear what you suggest. Could you please clarify ?

Reply:
fixed